cmd/lakectl/cmd/root.go

```diff
@@ -72,10 +76,11 @@ type Configuration struct {
 		Provider        struct {
 			Type   lakefsconfig.OnlyString `mapstructure:"type"`
 			AWSIAM struct {
-				TokenTTL            time.Duration      `mapstructure:"token_ttl_seconds"`
-				URLPresignTTL       time.Duration      `mapstructure:"url_presign_ttl_seconds"`
-				RefreshInterval     time.Duration      `mapstructure:"refresh_interval"`
-				TokenRequestHeaders *map[string]string `mapstructure:"token_request_headers"`
+				TokenTTL                   time.Duration      `mapstructure:"token_ttl_seconds"`
+				URLPresignTTL              time.Duration      `mapstructure:"url_presign_ttl_seconds"`
+				RefreshInterval            time.Duration      `mapstructure:"refresh_interval"`
+				TokenRequestHeaders        *map[string]string `mapstructure:"token_request_headers"`
+				ClientLogPreSigningRequest bool               `mapstructure:"client_log_pre_signing_request"`
```

Comment:
I don't think this parameter is relevant to lakectl.
It's only relevant to proccess that run in k8s AFAIK

Reply:
Sorry, I've got confused with a different flag we had.